api/src/util/text_extractor.py

```diff
@@ -131,30 +195,31 @@ def __init__(
         self.config = get_text_extractor_configs()[self.file_type]
 
     def get_text(self) -> str:
+        logger.info(f"{TEXT_EXTRACTOR_LOG_PREFIX} start: {self.file_path=} {self.file_type=}")
         try:
-            return self.config.extractor(self._read_file_data()).strip()
+            extracted_data = self.config.extractor(self._read_file_data()).strip()
+            logger.info(
+                f"{TEXT_EXTRACTOR_LOG_PREFIX} success: {self.file_path=} {self.file_type=} characters_extracted={len(extracted_data)}"
+            )
```

Comment:
Don't try to shove info into the log message, log messages should be kept generic, put it in the extra object so it's easily parsed out.